### ISSUE TITLE
Can't reproduce of the results

### ISSUE THREAD

**Issue by Amazingren** [jiexiaou/ShuffleFormer#2]
Dear @jiexiaou ,

Thanks for your impressive insight for the random shuffle operation in Image Restoration, which is really interesting.

However, when I tried to reproduce the results with the provided code, I could not reach the results reported in the paper(Denoising on SIDD).

![image](https://github.com/jiexiaou/ShuffleFormer/assets/31409716/47ec3c63-795c-47e6-b8e9-4989ae97127f)

To this end, I am reaching out to ask what's the possible reason caused such results. Or could you update the code when you are available?

I am looking forward to hearing from you soon.

Bests and have a nice day,

**Comment by jiexiaou** [comment]
Thanks for your attention! We have uploaded pretrained checkpoint for denoising on SIDD. And we will release the training code as soon as possible.

**Comment by Amazingren** [comment]
> Thanks for your attention! We have uploaded pretrained checkpoint for denoising on SIDD. And we will release the training code as soon as possible.

Thanks a lot for the quick feedback. I will try to test the model with the updated test file and the corresponding ckpts!
 
I am also looking forward to the training code! :)

Bests and many thanks,

**Comment by Quyangya** [comment]
When will we be able to release the Deblurring pre-trained model?

**Comment by Amazingren** [comment]
> When will we be able to release the Deblurring pre-trained model?

no idea yet
